Example/PaymentSheet Example/PaymentSheetUITest/PaymentSheetUITest.swift

```diff
@@ -2118,7 +2118,7 @@ class PaymentSheetDeferredServerSideUITests: PaymentSheetUITestCase {
     // MARK: - External PayPal 
     func testExternalPaypalPaymentSheet() throws {
         var settings = PaymentSheetTestPlaygroundSettings.defaultValues()
-        settings.externalPayPalEnabled = .on
+        settings.externalPaymentMethods = .paypal
```

Comment:
Didn't add external Venmo tests here as it seems like it would just be testing the same code path. Might be worth adding a test that uses multiple external payment methods e.g. `.both`?